windows_docker_resources/Dockerfile.foxy

```diff
@@ -28,8 +28,8 @@ ADD https://slproweb.com/download/Win64OpenSSL-1_0_2u.exe C:\TEMP\Win64OpenSSL.e
 # OpenCV
 ADD https://github.com/ros2/ros2/releases/download/opencv-archives/opencv-3.4.6-vc16.VS2019.zip C:\TEMP\opencv.zip
 
-# Python 3.7
-ADD https://www.python.org/ftp/python/3.7.6/python-3.7.6-amd64.exe C:\TEMP\python-37.exe
+# Python 3.8
+ADD https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe C:\TEMP\python-38.exe
```

Comment:
3.8.3 is out now. Do you feel lucky?

Reply:
:game_die: https://github.com/ros2/ci/pull/452